### PR TITLE
August 2019 edits

### PR DESCRIPTION
**Pull request by rudeboybert**

### CODE REVIEW

**Comment by rudeboybert** [comment]
@ismayc This PR ensures all figures and tables are correctly numbered. Although we won't be 100% sure unless we comb thru the PDF one more time (we can do this at proofreading). I was careful to break down all changes incrementally in the list of [commits](https://github.com/moderndive/moderndive_book/pull/176/commits). 

I figured I'd get these done ASAP (in time for the update of moderndive.com) and focus on text edits this weekend.

**Comment by ismayc** [comment]
Thanks! Will review as soon as I can. Potentially tomorrow.

File: 03-visualization.Rmd [hunk 1]
```diff
@@ -462,7 +462,7 @@ Let's consider the `temp` variable in the `weather` data frame once again, but u
 
 One way to visualize this *distribution* \index{distribution} of this single variable `temp` is to plot them on a horizontal line as we do in Figure \@ref(fig:temp-on-line):
 
-```{r temp-on-line, echo=FALSE, fig.height=0.8, fig.cap="Plot of Hourly Temperature Recordings from NYC in 2013"}
+```{r temp-on-line, echo=FALSE, fig.height=0.8, fig.cap="Plot of hourly temperature recordings from NYC in 2013"}
```
Comment (ismayc) [inline 03-visualization.Rmd]:
```suggestion
```{r temp-on-line, echo=FALSE, fig.height=0.8, fig.cap="Plot of hourly temperature recordings from NYC in 2013."}
```

File: 02-getting-started.Rmd [hunk 2]
```diff
@@ -413,9 +431,10 @@ We've given you what we feel are the most essential concepts to know before you
 
 If you are completely new to the world of coding, R, and RStudio and feel you could benefit from a more detailed introduction, we suggest you check out ModernDive co-author Chester Ismay's [Getting used to R, RStudio, and R Markdown](https://rbasics.netlify.com/) short book [@usedtor2016], which includes screencast recordings that you can follow along and pause as you learn. Furthermore, there is an introduction to R Markdown, a tool used for reproducible research in R.
 
-<center>
-![](images/getting-used-to-R.png){ height=3.5in }
-</center>
+```{r echo=FALSE, fig.align='center', fig.cap="Preview of Getting used to R, RStudio, and R Markdown.\"", out.height=if(knitr:::is_latex_output()) '2.5in', purl=FALSE}
```
Comment (ismayc) [inline 02-getting-started.Rmd]:
```suggestion
```{r echo=FALSE, fig.align='center', fig.cap="Preview of Getting used to R, RStudio, and R Markdown.", out.height=if(knitr:::is_latex_output()) '2.5in', purl=FALSE}
```

File: 06-regression.Rmd [hunk 3]
```diff
@@ -1082,7 +1082,7 @@ Any other line draw in the figure would yield a sum of squared residuals greater
 * An arbitrarily chosen line in dashed red
 * Another arbitrarily chosen line in dashed green
 
-```{r, echo=FALSE}
+```{r three-lines, fig.cap="Regression line and two others.", fig.height=3.5, echo = FALSE}
```
Comment (ismayc) [inline 06-regression.Rmd]:
```suggestion
```{r three-lines, fig.cap="Regression line and two others.", fig.height=3.5, echo=FALSE}
```

File: 06-regression.Rmd [hunk 4]
```diff
@@ -1120,10 +1120,14 @@ score_model <- lm(score ~ bty_avg, data = evals_ch6)
 # Get regression table:
 get_regression_table(score_model)
 ```
-```{r, echo = FALSE}
+```{r recall-table, echo = FALSE}
```
Comment (ismayc) [inline 06-regression.Rmd]:
```suggestion
```{r recall-table, echo=FALSE}
```

File: 06-regression.Rmd [hunk 5]
```diff
@@ -1145,7 +1149,7 @@ score_model %>%
   rename(lower_ci = conf_low,
          upper_ci = conf_high)
 ```
-```{r, echo = FALSE}
+```{r regtable-broom, echo = FALSE}
```
Comment (ismayc) [inline 06-regression.Rmd]:
```suggestion
```{r regtable-broom, echo=FALSE}
```

**Pull request merged**